### PR TITLE
Call app-mgt component to handle inbound details

### PR DESCRIPTION
**Pull request by sahandilshan**
## Purpose

With the https://github.com/wso2/carbon-identity-framework/pull/5146 improvement, we just need to call the app-mgt component to handle the inbound OAuth and SAML protocol details. We don't have to explicitly call each inbound admin service from the API level to handle those protocols.

## Approach
image

*** Note:
Framework, Inbound OAuth and SAML versions should be updated

### CODE REVIEW

**Comment by VivekVinushanth** [comment]
Build is failing without any changes and the core team is checking on it.

File: components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/inbound/passive/sts/PassiveSTSInboundFunctions.java [hunk 1]
```diff
@@ -91,4 +95,13 @@ public static PassiveStsConfiguration getPassiveSTSConfiguration(InboundAuthenti
         return new PassiveStsConfiguration().realm(inboundAuth.getInboundAuthKey()).replyTo(replyTo)
                 .replyToLogout(replyToLogout);
     }
+    
+    private static Map<String, Object> buildAuditLogData(PassiveStsConfiguration passiveSTSConfig) {
```
Comment (sahandilshan) [inline components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/inbound/passive/sts/PassiveSTSInboundFunctions.java:99]: Change into passiveSTS data

File: components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/pom.xml [hunk 2]
```diff
@@ -167,6 +172,10 @@
 
     <build>
         <plugins>
+<!--            <plugin>-->
```
Comment (piraveena) [inline components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/pom.xml:175]: why this is commented

File: components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/InboundProtocolToDTO.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+package org.wso2.carbon.identity.api.server.application.management.v1.core.functions.application;
```
Comment (piraveena) [inline components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/InboundProtocolToDTO.java:1]: license is missing

**Review by piraveena — APPROVED** [review]

**Pull request merged**